doc/source/loggers.rst

```diff
@@ -13,24 +13,106 @@ by caproto, below. Because recalling the name is not always convenient, certain
 objects in caproto's Python API expose a ``log`` attribute, such as ``pv.log``,
 ``pv.context.log`` and ``pv.circuit_manager.log``.
 
+You could find python logging flow here.
+https://docs.python.org/2/_images/logging_flow.png
```

Comment:
Let's link to this diagram on the page instead of directly to the image. I found it here: https://docs.python.org/3/howto/logging.html#logging-flow

Also, note that we should use link to `docs.python.org/3/... for Python 3 ` not `/2/...`.